releasenotes/notes/docupdate-291f410b433e5973.yaml

```diff
@@ -0,0 +1,4 @@
+---
+other:
+  - |
+    Documentation update for the GUI and system tray, and delayed service start
```

Comment:
no need for a release note for a doc update, could you remove it? (+ `noreno` tag)